modules/backend/behaviors/RelationController.php

```diff
@@ -356,6 +356,28 @@ public function initRelation($model, $field = null)
         $this->relationObject = $this->model->{$field}();
         $this->relationModel = $this->relationObject->getRelated();
 
+        /*
+         * Relationship Parent
+         */
+
+        $parentClass = get_class($this->model);
+        $allRelations = $this->relationModel->getRelationDefinitions();
+
+        $parentClassName = "";
+
+        foreach($allRelations as $relations)
+        {
+            foreach($relations as $key => $value)
+            {
+                if (strtolower($value[0]) == strtolower($parentClass))
+                    $parentClassName = $key;
+            }
+        }
+
+        if ($parentClassName != "")
+              $this->relationModel->{$parentClassName} = $this->model;
```

Comment:
Some codestyle issues in this block:
- Opening bracket should be on the same line on the foreach loops
- Missing opening and closing brackets on the single line if conditions